week02/assignment02_test.go

```diff
@@ -0,0 +1,109 @@
+package week02
+
+import "testing"
+
+func TestArray(t *testing.T) {
+	exp := [4]string{"max", "tuto", "bombom", "tony"}
+	act := make([]string, 0)
```

Comment:
Since your creating a zero length slice, you don't need to use `make`. Instead you can use either of the lines below to achieve the same result, and be idiomatic.

```suggestion
	act := []string{}
	var act []string
```